src/plugins/intel_cpu/src/transformations/transformation_pipeline.cpp

```diff
@@ -902,9 +902,14 @@ void Transformations::runLptPasses(const std::vector<ov::element::Type>& default
     ov::pass::Manager lptManager("CPU:LPT");
 
 #if defined(OPENVINO_ARCH_ARM) || defined(OPENVINO_ARCH_ARM64)
-    auto quantizationRestrictions = std::vector<QuantizationGranularityRestriction>();
+    auto quantizationRestrictions = std::vector<QuantizationGranularityRestriction>(
+        {QuantizationGranularityRestriction::create<ov::opset1::Convolution>({0}),
+         QuantizationGranularityRestriction::create<ov::opset1::ConvolutionBackpropData>({0})});
```

Comment:
Do we really need `QuantizationGranularityRestriction` and `PrecisionsRestriction` for `ConvolutionBackpropData`? It is not supported yet I suppose

Reply:
Removed `ConvolutionBackpropData` from these lists